src/Entity/HearingPost.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\HearingPostRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * @ORM\Entity(repositoryClass=HearingPostRepository::class)
+ */
+class HearingPost
+{
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="uuid", unique=true)
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=Hearing::class, inversedBy="hearingPosts")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $hearing;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=Party::class)
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $recipient;
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private $content;
+
+    /**
+     * @ORM\Column(type="boolean", options={"default":"0"})
+     */
+    private $hasBeenProcessedAndForwarded = false;
```

Comment:
Consider using a `?DateTimeInterface` value here.

Reply:
Fixed in 3d4a02477b062529bd25cb3e4c1e4fbc8041c8c9.